views/options/settings.jade

```diff
@@ -88,10 +88,30 @@ script(type='text/ng-template', id='partials/options.settings.settings.html')
           .panel-heading
             span Registration
           .panel-body
-            p(ng-if='user.auth.facebook.id') Registered with Facebook
+            p(ng-if='user.auth.facebook.id') 
+              =env.t('registeredWithFb')
             div(ng-if='user.auth.local.username')
-              p Username: {{user.auth.local.username}}
-              p Email: {{user.auth.local.email}}
+              p
+              =env.t('username') 
+              |: {{user.auth.local.username}}
+              p
+              small
```

Comment:
`p.small.muted` would give a paragraph tag the classes `small` and `muted`. What you want to do is indent the `small` tag so it's nested inside the `p` and give that the class of muted. So this:

``` jade
p
  small.muted
```

Make sense?